server/project/api/models.py

```diff
@@ -24,10 +53,32 @@ class Timezone(db.Model):
     minutes = db.Column(db.Integer)
     dst_hours = db.Column(db.Integer, nullable=False)
     dst_minutes = db.Column(db.Integer)
-    
+
     def __init__(self, name, hours, minutes, dst_hours, dst_minutes):
         self.name = name
         self.hours = hours
         self.minutes = minutes
         self.dst_hours = dst_hours
         self.dst_minutes = dst_minutes
+
+
+class Event(db.Model):
+    __tablename__ = 'events'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(32), nullable=False)
+    location = db.Column(db.String(256))
+    description = db.Column(db.String(1024))
+    url = db.Column(db.String(32), nullable=False, unique=True)
+    user_id = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=True)
```

Comment:
should events be allowed to not have a user?

Reply:
Fixed.